tests/structuredheadings/formatcombo-boundaries.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo,undo */
+/* eslint-enable */
+
+// Clean up all instances been created on the page.
+(function () {
+  var comboName = "NumFormats";
+
+  bender.editor = {
+    allowedForTests: "h1; h2; p"
+  };
+
+  bender.test({
+    setUp: function () {
+      //Anything to be run before each test if needed
+    },
+
+    tearDown: function () {
+      var editor = this.editorBot.editor;
+      editor.plugins.structuredheadings.currentScheme = "1.1.1.1.1.";
```

Comment:
The 1.1.1.1. is the default state, so this actually is a way of returning to default.  It can go in setUp as well, in this particular case.  

Because of weird timing issues (and the fact the editor is not reloaded between multiple test methods), it's required in one of the two.